docs/environment-variables.md

```diff
@@ -67,6 +67,15 @@ The OAuth access token, generated by Launchpad.
 
 The OAuth token secret, generated by Launchpad.
 
+
+## Ubuntu Store settings
+
+### STORE\_MACAROON\_API\_URL
+
+- **Example**: `https://myapps.developer.ubuntu.com/dev/api`
+
+The URL of the Ubuntu Store Macaroon API service.
```

Comment:
I'd call this just `STORE_API_URL` and "the Ubuntu Store API service", I think; there's more than just the macaroon API under there, and we might want some other bit of it in future.